community/customization/wings.md

```diff
@@ -0,0 +1,42 @@
+# Customizing Wings
+:::warning Production
+We highly recommend **NOT** performing these actions on production
+:::
+
+In order to make changes to wings we recommend forking it to keep your own git version making upgrades easier. <br />
```

Comment:
This is a confusing sentence at least in my head, it probably requires rephrasing or at least a comma. If I understood right, this is what is meant (not a suggestion to replace with that sentence):

"We recommend creating a fork of the repo before making changes to wings to make future upgrades easier"